apps/community/src/app/lab/components/LabCard.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import type { Lab } from '../mocks/labs';
+import * as styles from './LabCard.css';
+
+function LabCard({ lab }: { lab: Lab }) {
+  return (
+    <div className={styles.cardContainer}>
+      <Image
+        src={lab.img || 'https://placehold.co/128'}
```

Comment:
```suggestion
        src={lab.img ?? 'https://placehold.co/128'}
```

ret: https://ko.javascript.info/nullish-coalescing-operator

Reply:
nullish 병합 연산자는 처음 들어봤네요 .. 👍